src/Export/DynamicProductGroupService.php

```diff
@@ -218,10 +246,109 @@ private function buildCriteria(): Criteria
         return $criteria;
     }
 
-    private function getCacheItem(): CacheItemInterface
+    private function getExportPaginationCacheItem(): CacheItemInterface
```

Comment:
The name export pagination cache item, is a bit confusing. Can this be renamed so its clear what it is used for?